catalogue/webapp/components/WorkCard/WorkCard.js

```diff
@@ -0,0 +1,224 @@
+            [spacing({ s: 3 }, { padding: ['bottom', 'top'] })]: true,
+            'card-link': true,
+          })}
+          onClick={() => {
+            trackEvent({
+              category: 'WorkCard',
+              action: 'follow link',
+              label: work.id,
+            });
+          }}
+        >
+          <Container>
+            <Details>
+              <div
+                className={classNames({
+                  flex: true,
+                  'flex--v-center': true,
+                  [font({ s: 'HNL4' })]: true,
+                  [spacing({ s: 1 }, { margin: ['bottom'] })]: true,
+                })}
+              >
+                {workTypeIcon && (
+                  <Icon
+                    name={workTypeIcon}
+                    extraClasses={classNames({
+                      [spacing({ s: 1 }, { margin: ['right'] })]: true,
+                    })}
+                  />
+                )}
+                {work.workType.label}
+              </div>
+              <h2
+                className={classNames({
+                  [font({ s: 'HNM3' })]: true,
+                  'card-link__title': true,
+                })}
+              >
+                {work.title}
+              </h2>
+              <div
+                className={classNames({
+                  flex: true,
+                })}
+              >
+                {work.contributors.length > 0 && (
+                  <div
+                    className={classNames({
+                      [spacing({ s: 2 }, { margin: ['right'] })]: true,
+                    })}
+                  >
+                    <LinkLabels
+                      items={work.contributors.map(({ agent }) => ({
+                        text: agent.label,
+                        url: null,
+                      }))}
+                    />
+                  </div>
+                )}
+                {productionDates.length > 0 && (
+                  <LinkLabels
+                    heading={'Date'}
+                    items={productionDates.map(date => ({
+                      text: date,
+                      url: null,
+                    }))}
+                  />
+                )}
+              </div>
+            </Details>
+
+            <Preview
+              className={classNames({
+                [spacing({ s: 2 }, { margin: ['left'] })]: true,
+                'text-align-center': true,
+              })}
+              style={{
+                flexGrow: 0,
+                flexShrink: 0,
+                flexBasis: '178px',
+                height: '178px',
+              }}
+            >
+              {work.thumbnail && (
+                <img
+                  src={work.thumbnail.url}
+                  className={classNames({
+                    'h-center': true,
+                  })}
+                  style={{
+                    width: 'auto',
+                    height: 'auto',
+                    maxWidth: '100%',
```

Comment:
We have `img { width: 100% }` as a responsive default. Do we need this as well?

Reply:
Yip, as it can go bigger:
![screencast-localhost-3000-2019 02 26-12-37-17](https://user-images.githubusercontent.com/31692/53413215-5476f980-39c3-11e9-9df6-00c09a5cd2b9.gif)